cmd/kube-controller-manager/app/controllermanager_test.go

```diff
@@ -156,3 +158,17 @@ func TestFeatureGatedControllersShouldNotDefineAliases(t *testing.T) {
 		}
 	}
 }
+
+// TestTaintEvictionControllerDeclaration ensures that it is possible to run taint-manager as a separated controller
+// only when the SeparateTaintEvictionController feature is enabled
+func TestTaintEvictionControllerDeclaration(t *testing.T) {
```

Comment:
I think we can remove this test now, or just check the presence of `descriptor.requiredFeatureGates` if you really want to keep it.

Reply:
Same thing as the other comment.